app/styles/play/menu/game-menu-modal.sass

```diff
@@ -108,11 +108,11 @@
     .game-menu-tab-content
       position: absolute
       left: 85px
-      top: 190px
+      top: 100px
       background: #ffffff
       border-radius: 8px
       width: 478px
-      height: 500px
+      height: 570px
```

Comment:
Do we actually need the `height` to be set? Can we just remove the `height`, to let it grow if new items added? - And maybe set a `max-height` to prevent it from growing too large?